tests/extract/conftest.py

```diff
@@ -0,0 +1,207 @@
+try:  # AVOID isort
```

Comment:
I believe you did this while doing development but you don't need it anymore.
Is this correct?

Reply:
This is required for production:  `isort` demands imports be in alphabetical order, but `    from treeherder.extract import extract_jobs` must be first to setup the environment.